www/js/shortcut_tray.json

```diff
@@ -2,8 +2,8 @@
     "anandSahib": {
         "label": "Anand Sahib",
```

Comment:
just wonder, should we some how specify this is 6 pauri anand sahib (with salok)?

Reply:
cc. @maneetpaul

Reply:
@inderpreetsingh can we label it "Anand Sahib + Salok"